content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -182,6 +182,7 @@ These instructions are for Kubernetes {{< skew currentVersion >}}.
    The same signing key is used for all repositories so you can disregard the version in the URL:
 
    ```shell
+   mkdir -p /etc/apt/keyrings
```

Comment:
We should do one of:
- Move the note (line 208) here, and include a comment such as:
  ```suggestion
     # If you see an error, read the following note
  ```

- ```suggestion
     sudo mkdir -m 755 /etc/apt/keyrings
  ```
  (copied from line 208)
  For this option, let's also add a comment about _why_ this is needed (older Debian versions don't have that directory).

Reply:
updated